tapdb/sqlite.go

```diff
@@ -168,15 +168,21 @@ func (s *SqliteStore) ExecuteMigrations(target MigrationTarget) error {
 // NewTestSqliteDB is a helper function that creates an SQLite database for
 // testing.
 func NewTestSqliteDB(t *testing.T) *SqliteStore {
-	t.Helper()
-
-	dbFileName := filepath.Join(t.TempDir(), "tmp.db")
-	t.Logf("Creating new SQLite DB for testing: %s", dbFileName)
-
 	// TODO(roasbeef): if we pass :memory: for the file name, then we get
 	// an in mem version to speed up tests
+	dbPath := filepath.Join(t.TempDir(), "tmp.db")
```

Comment:
I think the `t.Helper()` method is required in every helper method in the call stack to avoid it being attributed as the source of a test failure.